bin/circleci/build-frontend.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 if [[ " $TESTPILOT_ENABLE_PONTOON_BRANCHES " =~ " $CIRCLE_BRANCH " ]]; then
-    NODE_ENV=production ENABLE_PONTOON=1 npm run static
+    NODE_ENV=production ENABLE_PONTOON=1 ENABLE_DEV_LOCALES=1 npm run static
```

Comment:
Would it be better to just have a dev flag that governs each of these? I can't imagine they'll ever be out of sync.